plugins/repository-s3/qa/amazon-s3/src/test/java/org/elasticsearch/repositories/s3/AmazonS3Fixture.java

```diff
@@ -72,7 +72,8 @@ public static void main(String[] args) throws Exception {
             // Emulates S3
             final String storageUrl = "http://" + addressAndPort;
             final AmazonS3TestServer storageTestServer = new AmazonS3TestServer(storageUrl);
-            storageTestServer.createBucket(args[1]);
+            storageTestServer.createBucket("permanent_bucket_test");
+            storageTestServer.createBucket("temporary_bucket_test");
```

Comment:
Can we revert this please, and pass the bucket names as program args?

Reply:
We now pass the bucket names in on the command line again.